sbin/dhclient/dhclient.8

```diff
@@ -95,6 +95,9 @@ Forces
 .Nm
 to reject leases with unknown options in them.
 The default behaviour is to accept such lease offers.
+.It Fl n
+Don't wait for ARP resolution within
+.Nm
```

Comment:
Please keep the flags alphabetized by moving the added text to after line 84. Also the sentances explaining other flags are ending with a period. Thanks!